content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -69,15 +71,23 @@ export class _StartupOverlay extends React.PureComponent {
   }
 
   onSubmit() {
-    this.props.dispatch(ac.UserEvent({event: "SUBMIT_EMAIL"}));
+    this.props.dispatch(ac.UserEvent(Object.assign({event: "SUBMIT_EMAIL"}, this._getFormInfo())));
```

Comment:
`{event: "SUBMIT_EMAIL", ...this._getFormInfo()}` is fine here

Reply:
Out of curiosity, what is the main benefit of this form compared to Object.assign?

Reply:
no benefit, personally I just think it's easier to read :)

Reply:
it's also more hip, but I tried to stay consistent with the repo

Reply:
(just to clarify that doesn't mean your repo ain't hip)